### DISCUSSION TITLE
[Feature Request] Display Profiles - different configurations for different screens

### DISCUSSION THREAD

**Discussion by LaurDr** [rxhanson/RectanglePro-Community#616] [Ideas]
I use 3 screens, my Mac screen, an Ultrawide and a 16:9 one in a vertical layout. I need key bindings for thirds, half,  lower and upper half.  

Different profiles could allow users to re-use keybindings for different layouts.

F.e.: 
- id like cmd+opt+← to occupy the left half of my Mac screen and the left third on my ultra wide, while not doing anything on my vertical screen.
     - The first 2 cases can be achieved through "cycle sizes on half actions" but its kinda clunky  and it doesn't solve the issue with the vertical screen.
- On my vertical screen on the other hand, id like cmd + opt + ↑ to not maximize, but take over the upper half. 

This way I can reuse cmd + opt + →↑←↓ and keep it simple, instead of introducing 3 different variations, essentially having 12 commands.

**Comment by rxhanson** [comment]
Thanks for the detailed feedback! In thinking through how the configuration for this would be implemented in the settings window, where would you expect to look to find it? The main challenge is keeping the complexity of the Settings window down, so I'm open to ideas here.
